code/raw_code/01_clean_data.R

```diff
@@ -59,114 +62,138 @@
+  
+  # Data Cleaning Task #12
+  duplicates_by_fields <- raw_df_clean_deduped %>% 
+                            distinct_at(vars(-Record_ID, -Timestamp, -Email_Masked,
+                                             -Main_Street, -Nearest_Landmark, -Routes,
+                                             -Direction, -Cleanliness, -Line_of_Sight, 
+                                             -Lighting, -Obstacles, -Obstacle_Desc,
+                                             -Behavior, -On_Site_Survey, -Additional_Comments), .keep_all = TRUE) %>%
+                            arrange(Stop_ID, Timestamp, Email_Masked) %>%
+                            mutate(Duplicate = ifelse((Stop_ID == dplyr::lag(Stop_ID) | Stop_ID == dplyr::lead(Stop_ID)), "Duplicate","")) %>%
+                            dplyr::filter(Duplicate != "") %>%
+                            select(-Duplicate)
 
   
   #write.csv(duplicates_by_fields, here("data", "raw_data", "validation", "Duplicate Records by Stop ID Round 2.csv"), row.names = FALSE)
   
   # The following code block takes the manually inputted fields from the "Duplicate Records by Stop ID and Fields" sheet and assigns them to the
   # corresponding field in the original data set.
   raw_df_clean_deduped <- raw_df_clean_deduped %>% 
-                            left_join(duplicate_records_reconciled_df, by = c("Record_ID")) %>%
-                            mutate(Seating = ifelse(!is.na(Seating_Reconciled), Seating_Reconciled, Seating),
-                                   Shelter = ifelse(!is.na(Shelter_Reconciled), Shelter_Reconciled, Shelter),
-                                   Trash_Can = ifelse(!is.na(Trash_Can_Reconciled), Trash_Can_Reconciled, Trash_Can),
-                                   Wayfinding = ifelse(!is.na(Wayfinding_Reconciled), Wayfinding_Reconciled, Wayfinding),
-                                   Wayfinding_Accessibility = ifelse(!is.na(Wayfinding_Accessibility_Reconciled), Wayfinding_Accessibility_Reconciled, Wayfinding_Accessibility),
-                                   Sidewalk = ifelse(!is.na(Sidewalk_Reconciled), Sidewalk_Reconciled, Sidewalk),
-                                   Boarding_Area = ifelse(!is.na(Boarding_Area_Reconciled), Boarding_Area_Reconciled, Boarding_Area),
-                                   Crosswalks = ifelse(!is.na(Crosswalks_Reconciled), Crosswalks_Reconciled, Crosswalks),
-                                   Crosswalk_Features = ifelse(!is.na(Crosswalk_Features_Reconciled), Crosswalk_Features_Reconciled, Crosswalk_Features)) %>%
-                            select(-Seating_Reconciled, -Shelter_Reconciled, -Trash_Can_Reconciled,
-                                   -Wayfinding_Reconciled, -Wayfinding_Accessibility_Reconciled, -Sidewalk_Reconciled,
-                                   -Boarding_Area_Reconciled, -Crosswalks_Reconciled, -Crosswalk_Features_Reconciled)
+                              left_join(duplicate_records_reconciled_df, by = c("Record_ID")) %>%
```

Comment:
The extra indents are not consistent with above assignments to `raw_df_clean_deduped`, are they needed?

Reply:
Updated indentation for all dataframe assignments to be uniform